geode-core/src/main/java/org/apache/geode/internal/cache/RegionPerfStats.java

```diff
@@ -324,6 +347,16 @@ public void endGet(long start, boolean miss) {
     cachePerfStats.endGet(start, miss);
   }
 
+  @Override
+  public void endGetForClient(long start, boolean miss) {
+    long totalNanos = clock.isEnabled() ? getTime() - start : 0;
```

Comment:
I am sad that we are observing clock disablement here, but I understand that changing it would require too much work for this PR.

Reply:
One option I considered was always calling `clock.getTime()` instead of only calling it if the clock is enabled. This would work for now because the implementation returns zero if the clock is disabled. However, the StatisticsClock API doesn't specify that implementations of getTime() will always return zero if the clock is disabled. I suppose we could change the API since it's internal...